tooling/nargo_fmt/src/visitor/expr.rs

```diff
@@ -39,6 +39,18 @@ impl FmtVisitor<'_> {
                     self.format_expr(infix.rhs)
                 )
             }
+            ExpressionKind::Constructor(constructor_expr) => {
+                let type_str = constructor_expr.type_name.to_string();
+                let formatted_fields = constructor_expr
+                    .fields
+                    .iter()
+                    .map(|(field_ident, field_value)| {
+                        format!("{}: {}", field_ident, self.format_expr(field_value.clone()))
+                    })
+                    .collect::<Vec<_>>()
```

Comment:
It may be useful to add the iter-extended utility crate we have so that we could use `vecmap` here